modules/core/src/main/scala/org/scalasteward/core/git/package.scala

```diff
@@ -21,8 +21,8 @@ import org.scalasteward.core.repoconfig.CommitsConfig
 import org.scalasteward.core.update.show
 
 package object git {
-  def branchFor(update: Update): Branch =
-    Branch(s"update/${update.name}-${update.nextVersion}")
+  def branchFor(update: Update, baseBranch: Branch): Branch =
+    Branch(s"${baseBranch.name}#update/${update.name}-${update.nextVersion}")
```

Comment:
I see that you are replicating the format from the `repo.md` file here. But in my opinion 
`s"update/${baseBranch.name}/${update.name}-${update.nextVersion}"` reads more natural.